### PR TITLE
Add docs on trajectory concatenation

### LINKED ISSUE #1338: Document Trajectory Concatenation

### PR DESCRIPTION
**Pull request by Daltz333**
Closes #1338

### CODE REVIEW

**Review by prateekma — CHANGES_REQUESTED** [review]

File: source/docs/software/advanced-controls/trajectories/trajectory-generation.rst [hunk 1]
```diff
@@ -63,3 +63,48 @@ Here is an example of generating a trajectory using clamped cubic splines for th
...
+
+         frc::Trajectory m_trajectoryOne = frc::TrajectoryGenerator::GenerateTrajectory(
+            frc::Pose2d(0_m, 0_m, 0_rad),
+            {frc::Translation2d(1_m, 1_m), frc::Translation2d(2_m, -1_m)},
+            frc::Pose2d(3_m, 0_m, 0_rad), frc::TrajectoryConfig(3_fps, 3_fps_sq));
+
+         frc::Trajectory m_trajectoryTwo = frc::TrajectoryGenerator::GenerateTrajectory(
+            frc::Pose2d(3_m, 0_m, 0_rad),
+            {frc::Translation2d(4_m, 4_m), frc::Translation2d(5_m, 3_m)},
+            frc::Pose2d(6_m, 0_m, 0_rad), frc::TrajectoryConfig(3_fps, 3_fps_sq));
```
Comment (calcmogul) [inline source/docs/software/advanced-controls/trajectories/trajectory-generation.rst:108]:
```suggestion
         auto m_trajectoryOne = frc::TrajectoryGenerator::GenerateTrajectory(
            frc::Pose2d(0_m, 0_m, 0_rad),
            {frc::Translation2d(1_m, 1_m), frc::Translation2d(2_m, -1_m)},
            frc::Pose2d(3_m, 0_m, 0_rad), frc::TrajectoryConfig(3_fps, 3_fps_sq));

         auto m_trajectoryTwo = frc::TrajectoryGenerator::GenerateTrajectory(
            frc::Pose2d(3_m, 0_m, 0_rad),
            {frc::Translation2d(4_m, 4_m), frc::Translation2d(5_m, 3_m)},
            frc::Pose2d(6_m, 0_m, 0_rad), frc::TrajectoryConfig(3_fps, 3_fps_sq));
```

File: source/docs/software/advanced-controls/trajectories/trajectory-generation.rst [hunk 2]
```diff
@@ -47,13 +47,13 @@ Here is an example of generating a trajectory using clamped cubic splines for th
...
 
-   .. tab:: Java
+   .. tab:: java
 
       .. literalinclude:: examples/trajectory-generation-1/java/ExampleTrajectory.java
          :language: java
          :lines: 10-32
 
-   .. tab:: C++
+   .. tab:: cpp
```
Comment (prateekma) [inline source/docs/software/advanced-controls/trajectories/trajectory-generation.rst:56]: Why were these changed?

Comment (Daltz333) [reply source/docs/software/advanced-controls/trajectories/trajectory-generation.rst:56]: I was debugging why my build earlier was failing. Technically the previous syntax was wrong, and should be using group-tabs.

File: source/docs/software/advanced-controls/trajectories/trajectory-generation.rst [hunk 3]
```diff
@@ -63,3 +63,44 @@ Here is an example of generating a trajectory using clamped cubic splines for th
 
 
 .. note:: Even though this trajectory generation is orders of magnitude faster than Pathfinder, it is highly recommended to generate all trajectories on startup (``robotInit``) as the generation time is still not negligible. Generation time often ranges from 10 ms to 25 ms for each trajectory.
+
+Concatenating Trajectories
+--------------------------
+
+Trajectories in Java can be combined into a single trajectory using the ``concatenate(trajectory)`` function. C++ users can simple add (``+``) the two trajectories together.
```
Comment (prateekma) [inline source/docs/software/advanced-controls/trajectories/trajectory-generation.rst:70]: simple -> simply

**Review by prateekma — APPROVED** [review]

**Review by calcmogul — DISMISSED** [review]

**Review by jasondaming — CHANGES_REQUESTED** [review]
Can we add a comment about how this doesn't match up start/end velocities.  Could add it on to the warning that is there?

**Review by jasondaming — APPROVED** [review]
